### PR TITLE
[ENG-680] Inject HTTP body

### PR DESCRIPTION
**Pull request by tebeka**
This is a temporary fix to inject HTTP request body to Python.
We'll remove this once ENG-624 is implemented.

Fixes ENG-680

### CODE REVIEW

File: runtimes/pythonrt/Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ test-py:
 
 test-go:
 	golangci-lint run ./...
-	go test -v
+	go test $(SHORT) -v
```
Comment (itayd) [inline runtimes/pythonrt/Makefile]: where is this coming from?

File: runtimes/pythonrt/ak_runner.py [hunk 2]
```diff
@@ -379,5 +382,14 @@ def module_entries(mod):
     event = message.get('event')
     event = {} if event is None else event
 
+    # Inject HTTP body
+    # TODO (ENG-624) change this once we support callbacks to autokitteh
+    body = event.get('data', {}).get('body')
```
Comment (itayd) [inline runtimes/pythonrt/ak_runner.py:387]: oh wow this is very specific in a very generic place

**Review by itayd — APPROVED** [review]
super hacky, but will do in the meantime

File: runtimes/pythonrt/Makefile (hunk 1 above)
Comment (tebeka) [reply runtimes/pythonrt/Makefile]: You can set environment variable to run only short tests, this disables `Test_createVEnv`  which takes a long time.
Changed to `TESTOPTS` and added a comment.

File: runtimes/pythonrt/ak_runner.py (hunk 2 above)
Comment (tebeka) [reply runtimes/pythonrt/ak_runner.py:387]: Yes, I'll re-do it once we have stabler event API to Python.

**Pull request merged**